### PR TITLE
Add documentation for aggregations usage in advancedSearch

### PR DESCRIPTION
**Pull request by dbengsch**
Still to be done: documentation for swift and objective-C

### CODE REVIEW

File: source/includes/_kuzzleDataCollection.md [hunk 1]
```diff
@@ -342,6 +371,7 @@ Resolves to a `JSON object` containing:
 
 - the total number of matched documents
 - an `array` of `KuzzleDocument` objects
+- an `array` of `aggregations` objects if some are provided in the request (see the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.0/search-aggregations.html) for more details)
```
Comment (stafyniaksacha) [inline source/includes/_kuzzleDataCollection.md]: Why the es 5.x version ?

Comment (dbengsch) [inline source/includes/_kuzzleDataCollection.md]: I thought we were migrating to 5 in a few days right ?

Comment (scottinet) [inline source/includes/_kuzzleDataCollection.md]: I agree with @stafyniaksacha: I think we should stick with 2.x until we truly migrated to ES 5, in order to maintain consistency throughout the documentation.

Comment (dbengsch) [inline source/includes/_kuzzleDataCollection.md]: Migration in PR

**Pull request merged**